setup.py

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env python
-
-from distutils.core import setup
-import etrsitrs
-
-setup(name='etrs-itrs',
-      version=etrsitrs.__version__,
```

Comment:
This caused pip / setup.py to fail on egg_info generation due to etrsitrs requiring numpy. Numpy will only be installed by pip / setup.py in phases AFTER the egg_info phase. On any system without numpy currently installed etrsitrs installation would have raised an `no module numpy` exception during installation.